ci/requirements/notebooktests.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - coverage
   - dask
   - deprecation
-  - gdal
+  - gdal=3.8
```

Comment:
I think I've addressed the gdal issues over at my PR. You might try it out.

Reply:
The branch is here now: https://github.com/kmuehlbauer/wradlib/tree/update-ci-safe

Reply:
yeah indeed I saw that, better than just pinning to an old version